docs/references/javascript/organization/members.mdx

```diff
@@ -1,42 +1,105 @@
 
 ## `getMemberships()`
 
+Retrieves the list of memberships for the currently active organization.
+
 ```typescript
-function getMemberships(params?: GetMembersParams): Promise<PaginatedResponse<OrganizationMembership>>;
+function getMemberships(params?: GetMembersParams): Promise<ClerkPaginatedResponse<OrganizationMembership>>;
 ```
 
-Retrieve the members of the currently active organization.
-
 ### `GetMembersParams`
 
 | Name | Type | Description |
 | --- | --- | --- |
 | `initialPage?` | `number` | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. |
 | `pageSize?` | `number` | A number that indicates the maximum number of results that should be returned for a specific page. |
-| `role` | `string` | The [roles](/docs/organizations/roles-permissions) of memberships that will be included in the response. |
+| `role?` | <code>[`OrganizationCustomRoleKey`](#organization-custom-role-key)[]</code> | The [roles](/docs/organizations/roles-permissions) of memberships that will be included in the response. |
 
-<Callout type="danger">
-  `GetMembershipsParams` was updated to `GetMembersParams` in September, 2023. The `limit` and `offset` parameters are now deprecated.
-</Callout>
+#### `OrganizationCustomRoleKey`
 
+`OrganizationCustomRoleKey` is a string that represents the user's role in the organization. Clerk provides the [default roles](/docs/organizations/roles-permissions#default-roles) `org:admin` and `org:member`. However, you can create [custom roles](/docs/organizations/create-roles-permissions) as well.
 
-### Returns
+### `getMemberships()` returns
 
 | Type | Description |
 | --- | --- |
-| <code>Promise\<[PaginatedResponse][pag-ref]\<[OrganizationMembership][org-mem-ref]\>></code> | This method returns a `Promise` which resolves with a [`PaginatedResponse`][pag-ref] of [`OrganizationMembership`][org-mem-ref] objects. |
+| `Promise<ClerkPaginatedResponse<OrganizationMembership>>` | This method returns a `Promise` that resolves to a [`ClerkPaginatedResponse`][pag-ref] of [`OrganizationMembership`][org-mem-ref] objects. |
+
+### `getMemberships()` example
+
+<InjectKeys>
+
+<CodeBlockTabs type="npm-script" options={['NPM module', '<script>']}>
+```js filename="main.js"
+import Clerk from '@clerk/clerk-js';
+
+// Initialize Clerk with your Clerk publishable key
+const clerk = new Clerk('{{pub_key}}');
+await clerk.load();
+
+if (clerk.user) {
+  await clerk.organization.getMemberships()
+    .then((res) => console.log(res))
+    .catch((error) => console.log("An error occurred:", error.errors));
+} else {
+  document.getElementById("app").innerHTML = `
+    <div id="sign-in"></div>
+  `;
+
+  const signInDiv = document.getElementById("sign-in");
+
+  clerk.mountSignIn(signInDiv);
+}
+```
+
+```html filename="index.html"
```

Comment:
When I run the `<script>` example I get this error:
```
TypeError: Cannot read properties of null (reading 'getMemberships')
```
I do have an organization in my clerk app, and the user I'm signed in as is part of that organization. Is there something else I may be missing here? I checked in the browser console and my user object is defined, but Clerk.organization is null

Reply:
Maybe my organization is not active? But I'm not sure how to fix that